sanitizer-macros/src/sanitizer.rs

```diff
@@ -89,6 +181,8 @@ impl Display for SanitizerError {
             5 => "Invalid sanitizer",
             6 => "Wrong number of arguments",
             7 => "The argument can be only 64 bit int",
+            8 => "Enums can contain only unnamed field",
+            9 => "Only Option<T> supported for now",
```

Comment:
That would allow custom errors and it'll become harder to debug, it'll act like a panic, if we have error codes for fixed values of errors we can easily search them up, I think it's best we map numbers to error messages. It'll also act as a single place for all the error messages so it's easier to edit them for better clarity too